tests/unit/test_estimator.py

```diff
@@ -169,7 +169,20 @@
     "mpi": {"enabled": True, "custom_mpi_options": "options", "processes_per_host": 2}
 }
 DISTRIBUTION_SM_DDP_ENABLED = {
-    "smdistributed": {"dataparallel": {"enabled": True, "custom_mpi_options": "options"}}
+    "smdistributed": {"dataparallel": {"enabled": True, "custom_mpi_options": "options"}},
+    "torch_distributed": {"enabled": False}
```

Comment:
It seems correct to me. In `DISTRIBUTION_SM_DDP_DISABLED ` dataparallel property is removed since its defaulted to false if not in the distribution object.